clients/src/main/java/org/apache/kafka/common/protocol/Protocol.java

```diff
@@ -386,15 +402,31 @@
                                                                                         new ArrayOf(INT64),
                                                                                         "A list of offsets."));
 
+    public static final Schema LIST_OFFSET_RESPONSE_PARTITION_V1 = new Schema(new Field("partition",
+                                                                                        INT32,
+                                                                              "Topic partition id."),
```

Comment:
nit: Can you fix alignment? I know the convention here is a little annoying to follow, but it does make it easy to read the schemas.